src/components/templates/News.tsx

```diff
@@ -1,30 +1,35 @@
-import React, {useCallback} from 'react';
-import {Button, Text, TextType} from '../atoms';
-import {Box} from "@material-ui/core";
-import {useStore} from '../../store/storeConfig'
-import RootStore from '../../store/root.store'
-import {observer} from 'mobx-react-lite';
-
+import React, { useCallback } from "react";
+import { Button, Text, TextType } from "../atoms";
+import { Box } from "@material-ui/core";
+import { useStore } from "../../store/storeConfig";
+import RootStore from "../../store/root.store";
+import { observer } from "mobx-react-lite";
 
 const News: React.FC = () => {
-  const store: RootStore = useStore();
-  
-  const onNewsClick = useCallback((id: number) => {
-    store.selectNewsFlash(id)
-  }, [store]);
-  
-  return (<Box flexDirection={'column'}>
-    {store.newsFlashCollection.map((news) => {
-      const date :null|string = news.date ==null ? '': new Date(news.date).toLocaleDateString();
-      return <Button key={news.id} onClick={onNewsClick.bind(null, news.id)}>
-        <Box border={1} borderColor='#6495ed'>
-          <Text type={TextType.NEWS_FLASH_TITLE} children={`${date}, ${news.source}` }/>
-          <Text type={TextType.NEWS_FLASH_CONTENT} children={news.title}/>
-        </Box>
-      </Button>
-    })}
-  </Box>);
-};
+	const store: RootStore = useStore();
 
+	const onNewsClick = useCallback(
+		(id: number) => {
+			store.selectNewsFlash(id);
+		},
+		[store]
+	);
+
+	return (
+		<Box flexDirection={"column"}>
+			{store.newsFlashCollection.map(news => {
+				const date: null | string = news.date == null ? "" : new Date(news.date).toLocaleDateString();
```

Comment:
The values here could either be a date object (which is truthy) or null (which is falsy).
You could skip the comparison and simply do:
`news.date ? new Date(news.date).toLocaleDateString : ''`

When you DO use comparisons, use `===` unless you have a very good reason not to.